mne_bids/read.py

```diff
@@ -327,23 +327,24 @@ def _handle_info_reading(sidecar_fname, raw, verbose=None):
                         'the MEG JSON sidecar file corresponds to the raw '
                         'data for KIT files.')
         else:
-            hpi_freqs_json = sidecar_json['HeadCoilFrequency']
-            try:
-                hpi_freqs_raw, _, _ = mne.chpi.get_chpi_info(raw.info)
-            except ValueError:
-                logger.info(
-                    'Cannot verify that the cHPI frequencies provided in '
-                    'the MEG JSON sidecar file correspond to those in the '
-                    'raw data. (Was it converted from another format?)'
-                )
-            else:
-                if not np.allclose(hpi_freqs_json, hpi_freqs_raw):
-                    raise ValueError(
-                        f'The cHPI coil frequencies in the sidecar file '
-                        f'{sidecar_fname}:\n    {hpi_freqs_json}\ndiffer from'
-                        f' what is stored in the raw data:\n'
-                        f'    {hpi_freqs_raw}\nCannot proceed.'
+            if 'HeadCoilFrequency' in hpi_freqs_json:
+                hpi_freqs_json = sidecar_json['HeadCoilFrequency']
+                try:
+                    hpi_freqs_raw, _, _ = mne.chpi.get_chpi_info(raw.info)
+                except ValueError:
+                    logger.info(
+                        'Cannot verify that the cHPI frequencies provided in '
+                        'the MEG JSON sidecar file correspond to those in the '
+                        'raw data. (Was it converted from another format?)'
                     )
+                else:
+                    if not np.allclose(hpi_freqs_json, hpi_freqs_raw):
+                        raise ValueError(
+                            f'The cHPI coil frequencies in the sidecar file '
+                            f'{sidecar_fname}:\n    {hpi_freqs_json}\ndiffer from'
+                            f' what is stored in the raw data:\n'
+                            f'    {hpi_freqs_raw}\nCannot proceed.'
```

Comment:
We have had this conversion multiple times but to me the bids files take precedence. So it should not crash but overwrite the values in the native raw file. Cc @sappelhoff @hoechenberger

Reply:
Yes I agree that we should make this adjustment here too: sidecar values should take precedence.
@fravona2211 What you'd need to do is call `warn()` instead of raising an exception. Let me know if you need help!

Reply:
Hi! So do you simply want me to replace `raise ValueError()` with `warn()` at this part in the code?:)